### PR TITLE
update documentation for triple slash comments

### PR DESCRIPTION
**Pull request by vicancy**
According to https://github.com/dotnet/docfx/pull/1178

### CODE REVIEW

**Comment by superyyrrzz** [comment]
:shipit:

File: Documentation/spec/triple_slash_comments_spec.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Top level block tags are transformed to corresponding metadata in .NET data model.
+
+| Tags            | Metadata name      | Type
+| ---             | ---                | ---
+| `summary`      | `summary`| `string`
+| `remarks`      | `remarks`| `string`
+| `returns`      | `returns` | `string`
+| `value`      | `returns` | `string`
+| `exception`      | `exception`| `List<`@"Microsoft.DocAsCode.DataContracts.ManagedReference.ExceptionInfo"`>`
+| `seealso`      | `seealso`| `List<`@"Microsoft.DocAsCode.DataContracts.ManagedReference.LinkInfo"`>`
```
Comment (ansyral) [inline Documentation/spec/triple_slash_comments_spec.md]: our code also implements top level `see`? although i have no idea how we should show it in the page

File: Documentation/spec/triple_slash_comments_spec.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+### Non-toplevel tags
+Non-toplevel tags transformed to HTML tags in DocFX.
+
+|Tags            | Transformed | Description
+|---             | ---| ---
+| `para`         | `<p></p>`
+| `b`            | `<strong></strong>`
+| `i`      | `em`
+| `see[@langword]`      | `<xref></xref>` | [langwordMapping.yml](https://github.com/dotnet/docfx/blob/27f7f55746dc48f0d7700205c52dff071b51427b/Documentation/langwordmapping/langwordMapping.yml) lists supported language keywords in `DocFX`. DocFX leverages [cross reference](docfx_flavored_markdown.md#cross-reference) to reference language keywords. You can disable default langword resolver and apply your customized one by calling `docfx build --xref yourLangword.yml --noLangKeyword`
+| `see[@href]`      | `<a></a>`
```
Comment (ansyral) [inline Documentation/spec/triple_slash_comments_spec.md]: also `see[@cref]`?

File: Documentation/spec/toc.md [hunk 3]
```diff
@@ -1,4 +1,6 @@
...
-# [.NET Metadata Specification](metadata_dotnet_spec.md)
-# [DocFx Flavored Markdown](docfx_flavored_markdown.md)
-# [DocFx Design Spec](docfx_design_spec.md)
\ No newline at end of file
+﻿# Metadata Specification
+## [General Metadata Specification](metadata_format_spec.md)
+## [.NET Metadata Specification](metadata_dotnet_spec.md)
+# [DocFX Flavored Markdown](docfx_flavored_markdown.md)
+# [Triple Slash Comments Support](triple_slash_comments_spec.md)
+#[DocFX Design Specification](docfx_design_spec.md)
```
Comment (hellosnow) [inline Documentation/spec/toc.md]: >DocFX [](start = 2, length = 5)

prepand one space "# [DocFX"?

File: Documentation/spec/triple_slash_comments_spec.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+| `list type="table"`      | `<table></table>`
+| `list type="bullet"`      | `<ul></ul>`
+| `list type="number"`      | `<ol></ol>`
+| `c`      | `<code></code>`
+| `code`      | `<pre><code></code></pre>`
+
+Custom tags
+-------
+### inheritdoc
+`docfx` supports a subset of the [inheritdoc functionality available in Sandcastle](https://ewsoftware.github.io/XMLCommentsGuide/html/86453FFB-B978-4A2A-9EB5-70E118CA8073.htm). Specifically, it implements most of the "Top-Level Inheritance Rules". It dose not implement:
```
Comment (pascalberger) [inline Documentation/spec/triple_slash_comments_spec.md]: Typo: dose

Comment (pascalberger) [inline Documentation/spec/triple_slash_comments_spec.md]: Typo: dose

File: Documentation/spec/triple_slash_comments_spec.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
+Triple-slash (///) Code Comments Support
+==========================================
+DocFX extracts [triple-slash (///) code comments](https://docs.microsoft.com/en-us/dotnet/articles/csharp/programming-guide/xmldoc/xml-documentation-comments) from .NET source code when running `docfx metadata`. [Tags](https://docs.microsoft.com/en-us/dotnet/articles/csharp/programming-guide/xmldoc/recommended-tags-for-documentation-comments) in triple-slash (///) code comments are converted to corresponding metadata in .NET data model.
+
+> [!NOTE]
+> `docfx` supports [DocFX Flavored Markdown syntax](docfx_flavored_markdown.md) inside triple-slash (///) code comments.
```
Comment (hellosnow) [inline Documentation/spec/triple_slash_comments_spec.md]: >supports [](start = 10, length = 8)

need we note we can skip mark up by command option "ShouldSkipMarkup"?

**Comment by hellosnow** [comment]
:shipit:

**Pull request merged**